src/apply-polyfill.ts

```diff
@@ -2,9 +2,9 @@ import {DEFAULT_POLYFILL_OPTIONS, getCompletePolyfill, getPolyfill, NavigationPo
 import {getNavigation} from "./get-navigation";
 import {globalNavigation} from "./global-navigation";
 
-export async function applyPolyfill(options: NavigationPolyfillOptions = DEFAULT_POLYFILL_OPTIONS) {
-  const { apply } = getCompletePolyfill(options);
-  await apply();
+export function applyPolyfill(options: NavigationPolyfillOptions = DEFAULT_POLYFILL_OPTIONS) {
+  const { apply } = getCompletePolyfill({ ...DEFAULT_POLYFILL_OPTIONS, ...options });
```

Comment:
That was probably the intention here? Otherwise providing `{ navigation }` (as is the case when importing `/polyfill`) will overwrite the defaults.

Reply:
When I read the options, I default there, so its not needed to do the same throughout like this. 

```typescript
  const {
    persist: PERSIST_ENTRIES,
    persistState: PERSIST_ENTRIES_STATE,
    history: givenHistory,
    limit: patchLimit,
    patch: PATCH_HISTORY,
    interceptEvents: INTERCEPT_EVENTS,
    window: givenWindow = globalWindow,
    navigation: givenNavigation
  } = {
    // These are super default options, if the object de
    ...DEFAULT_POLYFILL_OPTIONS,
    ...options
  }
```

If a value is not given in options, it is defaulted if its in those default options, if its in the options, it will override any default.